src/rnp/fficli.cpp

```diff
@@ -463,7 +466,7 @@ ffi_pass_callback_stdin(rnp_ffi_t        ffi,
             }
         }
     }
-    if (protect && is_primary) {
+    if ((protect | add_subkey) && is_primary) {
```

Comment:
Should not we use logical or here (i.e. `||`)?

Reply:
fixed